### ISSUE TITLE
IPv6 addresses display errors

### ISSUE THREAD

**Issue by gdiazlo** [wazuh/wazuh-kibana-app#5464]
| Wazuh | Elastic | Rev  | Security |
| ----- | ------- | ---- | -------- |
| 4.4.2   | -    | - | - |

| Browser |
| ------- |
| All |

**Description**
Across the application, all IPs can be reported as IPv4 and IPv6. We must adapt all IP fields to render appropriately.
We have two cases mainly: when an address does not fit in the column of a table and overflows to the next line, and when the IP does not fit in the container and overflows outside of it, mixing the text with other elements.

We must design a way to correctly render these IP addresses. If we decide to truncate the address, we must add a copy button, so the user can always access the complete address.

**Screenshots**
![image](https://github.com/wazuh/wazuh-kibana-app/assets/2181918/dfe7b816-078c-4526-9c97-11b16cf30348)

![image](https://github.com/wazuh/wazuh-kibana-app/assets/2181918/cde8a306-321a-437d-bbff-8829a398ff51)

**Comment by Desvelao** [comment]
TODO

@Tostti inform me in a meeting about the pending tasks of this issue.

He was researching how to implement the ellipsis for the IP address and rendering a button to copy the full value in the table of the Agents section.

I continued his research and I got something functional.

In the case of the Agents table:
For the ellipsis that could be displayed, it is required to remove the `tableLayout=true` property of the EuiTableBasic. This causes the width of the column to be not auto-sized, so the `ID` column would have a similar width to the `IP address` for example, when the content of the `ID` is significantly lower than another one.

The `tableLayout=true` property was added in this pull request https://github.com/wazuh/wazuh-kibana-app/pull/5143, because enhanced the space occupied by the columns. So, it doesn't seem a good idea to remove this property now.

Applying the ellipsis to the `IP address` column reduces the visual information that a user can see.

The `IP address` could be considered as important data of this column, so it could not make sense this data could be reduced through the ellipsis.

In the case we want to add the ellipsis to the `IP address` column, it needs to remove the `tableLayout=true` property.

Adding a button to copy the full value of the IP address for the cases when the value was truncated could have a high impact on the performance due to the needed calculations. This would have to be done for each IP address cell of the table.

- reduce the performance due to the required calculations to know if the element was truncated or not

# Research

**Comment by Desvelao** [comment]
We have discussed this issue today in the daily meeting.

For now, we have decided to allow the IP address value can truncate to the next line. This means the `truncateText` column property from the `IP address` columns should be `false` or undefined.

Regarding the agents table in the `Agents` section, @gdiazlo commented on the idea to redesign how the data is displayed. This could need researching the use cases and collaborating with the UX co-workers for the redesign.
